nifi-nar-bundles/nifi-enrich-bundle/nifi-enrich-processors/src/main/java/org/apache/nifi/processors/GeoEnrichIP.java

```diff
@@ -208,6 +120,8 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
             attrs.put(new StringBuilder(ipAttributeName).append(".geo.longitude").toString(), longitude.toString());
         }
 
+        attrs.put(new StringBuilder(ipAttributeName).append(".accuracy").toString(), response.getLocation().getAccuracyRadius().toString() );
```

Comment:
The javadoc for getAccuracyRadius() says "This attribute is only available from the Insights end point." Are we guaranteed for that to be the case? Perhaps add some defensive code here to only add the attribute if we can successfully get the radius (to avoid NPEs, e.g.)